.php_cs

```diff
@@ -107,10 +153,15 @@ $config->setRules(
         'single_line_comment_style' => true,
         'single_quote' => true,
         'space_after_semicolon' => true,
+        'standardize_increment' => true,
         'standardize_not_equals' => true,
+        'strict_comparison' => true,
+        'strict_param' => true,
+        'string_line_ending' => true,
```

Comment:
👎 This should be enforced by .editorconfig or something.

Reply:
For me, it's just a case of not liking having mixed line endings in a file - where we're enforcing a `LF` line ending for the actual code

Reply:
Which is why something else should ensure we all use the same line endings while inside a string having different ones can be desired, e.g. when preparing a message to send to some weird API.